libs/ledger-live-common/src/families/ethereum/bridge.integration.test.ts

```diff
@@ -53,6 +53,30 @@ const dataset: DatasetTest<Transaction> = {
                 },
               },
             },
+            {
+              name: "Send max",
+              transaction: (t) => {
+                return {
+                  ...t,
+                  mode: "send",
+                  gasPrice: new BigNumber(100),
+                  recipient: "0x17733CAb76d9A2112576443F21735789733B1ca3",
+                  useAllAmount: true,
+                  amount: new BigNumber(5000),
+                  feesStrategy: null,
+                };
+              },
+              expectedStatus: (account, tx, status) => {
+                return {
+                  amount: BigNumber.max(
+                    tx.amount.minus(status.estimatedFees),
```

Comment:
I'm not sure to understand why we have tx.amount and not account.spendable / balance